packages/base/src/panelview/objecttree.tsx

```diff
@@ -237,6 +237,27 @@ class ObjectTreeReact extends React.Component<IProps, IStates> {
     this.setState(old => ({ ...old, lightTheme }));
   };
 
+  handleNodeClick = (objectId: string) => {
+    const object = this.getObjectFromName(objectId);
+
+    if (object && object.visible === true) {
+      const objPosition = object?.parameters?.Placement?.Position || {
+        x: 0,
+        y: 0,
+        z: 0
+      };
+
+      const event = new CustomEvent('nodeClick', {
```

Comment:
I did not know one could define a custom event this way!

I guess we should be a tiny bit more specific about the event name. Maybe an event name like `jupytercadObjectSelection` would be better?

Reply:
Done! Thanks for suggesting.